PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -135,6 +135,7 @@
 
 _pp_on_AA_extraCommands = [
     'keep patPackedCandidates_hiPixelTracks_*_*',
+    'keep ints_hihffilter_*_*',
```

Comment:
hiHFfilter may be a bit more readable

Reply:
implemented in the commit b0ccac9877d896526f4a219906600508aa9cddef